src/connection-manager/connections.ts

```diff
@@ -167,6 +172,15 @@ async function createConnections(): Promise<void> {
     .whereNotNull('ip')
     .andWhere('start', '>', tenMinutesAgo)
 
+  const networksDb = await getNetworks()
+  networksDb.forEach((network) => {
+    nodes.push({
+      ip: network.entry,
+      ws: `ws://${network.entry}:51233`,
```

Comment:
You might want to add a port check in this, in case a port is mentioned

Reply:
The port in db is 51235, which is not what we want here. The findConnection function actually tried all ports in this list `[443, 80, 6005, 6006, 51233, 51234]` if we leave ws blank (line 149-155 of this file). Should I leave ws blank then?